firstvoices/backend/resources/media.py

```diff
@@ -45,10 +45,10 @@ class AudioResource(ControlledSiteContentResource):
         ),
     )
     exclude_from_kids = fields.Field(
-        column_name="audio_include_on_kids_site",
+        column_name="audio_include_in_kids_site",
         attribute="exclude_from_kids",
         widget=InvertedBooleanFieldWidget(
-            column="audio_include_on_kids_site", default=False
+            column="audio_include_in_kids_site", default=False
```

Comment:
The KB page specifies "on" -- was this changed intentionally?

Reply:
Yes, its slightly confusing. As per this [KB Page](https://firstvoices.atlassian.net/wiki/spaces/FIR1/pages/17664196/Prepare+batch+upload+spreadsheets+for+words+and+phrases#Example-batches)
In the dictionary entries,  its `INCLUDE_ON_KIDS_SITE` and `INCLUDE_IN_GAMES`.
while for media its `AUDIO_INCLUDE_IN_KIDS_SITE` and so on.

Reply:
+1

Reply:
Oh that's annoying. I logged a ticket to make them all match, but this is good for now. Thanks for explaining! https://firstvoices.atlassian.net/browse/FW-6546